packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -9,6 +9,7 @@
 import { AddPhoneNumberOptions } from '@azure/communication-calling';
 import { AudioDeviceInfo } from '@azure/communication-calling';
 import { AudioEffectsStartConfig } from '@azure/communication-calling';
+import { AzureCommunicationTokenCredential } from '@azure/communication-common';
```

Comment:
Looks like this is the class, use the interface `CommunicationUserIdentifier` instead